src/js/helpers/util.js

```diff
@@ -113,6 +113,30 @@ export const convertDateToQuarter = (date) => {
     return quarter;
 };
 
+export const convertQuarterToDate = (qtr, year) => {
```

Comment:
Is there a reason we have this?

Reply:
This was if we wanted the x-axis to reflect time instead of evenly spacing each submission, so it can be removed!